frontend/core-ui/src/modules/automations/components/builder/nodes/actions/manageProperties/component/ManageProperties.tsx

```
@@ -219,50 +204,35 @@
-            <Rule
-              key={index}
-              rule={rule}
-              handleChange={handleChange}
-              remove={handleRemove}
-              groups={groups}
-              propertyType={propertyType}
-              selectedField={selectedField}
-              operatorOptions={operators}
-            />
-          );
+          return <Rule {...updatedProps} />;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)